### ISSUE TITLE
Unable to update a product

### ISSUE THREAD

**Issue by Neil-github** [Davisonpro/advanced-rest-api#1]
Products can never be updated.

/controllers/v1/Product.php

function updateProduct

Line 217 -> 220

Currently, this code is always executed before the save is attempted : 

		return $api->response([
			'success' => false,
			'message' => 'Unable to update product'
		]);

The above 4 lines should be :  
		$product->name = $name;
		$product->description = $description;
		$product->price = (float) $price;
		$product->category_id = $category->id;

**Comment by sobhanmoghiseh** [comment]
@Neil-github You are right , but if this lines comes comment also product will be update .
